docs/EN_US/HPCC4J/HPCC4J_Mods/HPCC4J_Build.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="HPCC4J_Build">
+  <title>Buids</title>
+
+  <para>To build the projects using Maven, navigate to the base directory of
+  the project and issue the following command:</para>
+
+  <programlisting>mvn install</programlisting>
+
+  <para><variablelist>
```

Comment:
XML format doesn't seem consistent, elements likely should be new lined and indented

Reply:
XML formatting and indentation generated by our XML Editor application

Reply:
Fair enough. However, it feels wrong to allow a given editor to dictate the format style.